instrumentation/jetty/jetty-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v8_0/Jetty8HandlerAdvice.java

```diff
@@ -46,6 +55,17 @@ public static void stopSpan(
       @Advice.Local("otelContext") Context context,
       @Advice.Local("otelScope") Scope scope) {
 
-    JettyHandlerAdviceHelper.stopSpan(tracer(), request, response, throwable, context, scope);
+    if (scope == null) {
+      return;
+    }
+    scope.close();
+
+    if (throwable != null) {
+      if (response.getStatus() < 400) {
+        response.setStatus(500);
+      }
+    }
+
+    instrumenter().end(context, request, response, throwable);
```

Comment:
`JettyHandlerAdviceHelper.stopSpan` doesn't always end span (in case of async requests span will be ended from a listener). I suspect you really can't change that without breaking something.